src/bun.js/webcore/response.zig

```diff
@@ -4200,7 +4200,7 @@ pub const InternalBlob = struct {
     pub fn toJSON(this: *@This(), globalThis: *JSC.JSGlobalObject) JSValue {
         const str_bytes = ZigString.init(this.bytes.items).withEncoding();
         const json = str_bytes.toJSONObject(globalThis);
-        this.deinit();
+        // this.deinit();
```

Comment:
Why is this commented out? Wouldn't it cause the memory to never be freed?

Reply:
oh, this should be added back